tests/components/ecobee/test_number.py

```diff
@@ -0,0 +1,70 @@
+"""The test for the ecobee thermostat number module."""
+from unittest.mock import patch
+
+from homeassistant.components.number import ATTR_VALUE, DOMAIN, SERVICE_SET_VALUE
+from homeassistant.const import ATTR_ENTITY_ID, UnitOfTime
+
+from .common import setup_platform
+
+VENTILATOR_MIN_HOME_ID = "number.ecobee_ventilator_min_time_home"
+VENTILATOR_MIN_AWAY_ID = "number.ecobee_ventilator_min_time_away"
+
+
+async def test_ventilator_min_on_home_attributes(hass):
+    """Test the ventilator number on home attributes are correct."""
+    await setup_platform(hass, DOMAIN)
+
+    state = hass.states.get(VENTILATOR_MIN_HOME_ID)
+    assert state.state == "20"
+    assert state.attributes.get("min") == 0
+    assert state.attributes.get("max") == 60
+    assert state.attributes.get("step") == 5
+    assert state.attributes.get("friendly_name") == "ecobee Ventilator min time home"
+    assert state.attributes.get("unit_of_measurement") == UnitOfTime.MINUTES
+
+
+async def test_ventilator_min_on_away_attributes(hass):
+    """Test the ventilator number on away attributes are correct."""
+    await setup_platform(hass, DOMAIN)
+
+    state = hass.states.get(VENTILATOR_MIN_AWAY_ID)
+    assert state.state == "10"
+    assert state.attributes.get("min") == 0
+    assert state.attributes.get("max") == 60
+    assert state.attributes.get("step") == 5
+    assert state.attributes.get("friendly_name") == "ecobee Ventilator min time away"
+    assert state.attributes.get("unit_of_measurement") == UnitOfTime.MINUTES
+
+
+async def test_set_min_time_home(hass):
```

Comment:
I think this test is a bit week.
I think you should setup the platform and assert the state, then call the service and assert the state has changed to what you set in.

Reply:
Right now, in the code, I am waiting for a updated value from the web service to update the value. (aka, I don't store it locally) Should I do it differently?

Reply:
You can monkeypatch the fixture to alter the value and then move time so the update runs and get's the new number.
Example: https://github.com/home-assistant/core/blob/dev/tests/components/sensibo/test_number.py#L39-L49

Reply:
ok got it, done